plugin-server/src/worker/ingestion/property-definitions-manager.ts

```diff
@@ -74,6 +78,7 @@ export class PropertyDefinitionsManager {
     eventPropertiesCache: LRU<string, Set<string>> // Map<JSON.stringify([TeamId, Event], Set<Property>>
     eventLastSeenCache: LRU<string, number> // key: JSON.stringify([team_id, event]); value: parseInt(YYYYMMDD)
     propertyDefinitionsCache: PropertyDefinitionsCache
+    teamIdsToSkip: Set<number> = new Set()
```

Comment:
A value matcher would be slightly nicer so we can use "*" to rollout for all teams whilst still being able to roll back easily but not blocking

Reply:
Would rather delete the code once we make the switch - should be high confidence at that point, and the plugin server is enough of a mess already without also having all this stuff that's never used in prod but exists because it was once